tests/foreman/api/test_activationkey.py

```diff
@@ -173,14 +192,30 @@ def test_negative_update_1(self):
                 actual = {attr: getattr(act_key, attr) for attr in want.keys()}
                 self.assertEqual(want, actual)
 
-    def test_update_max_content_hosts(self):
+    def test_negative_update_invalid_name(self):
+        """@Test: Create activation key then update its name to an invalid name.
+
+        @Assert: Activation key is created, and its name is not updated.
+
+        @Feature: ActivationKey
+        """
+        new_key = entities.ActivationKey().create()
```

Comment:
If I change s/new/act then I'll have to drop the last assertion. Do you recommend that too @JacobCallahan ?